rholang/src/main/scala/coop/rchain/rholang/interpreter/RholangCLI.scala

```diff
@@ -50,20 +56,38 @@ object RholangCLI {
 
   private def printTask(normalizedTerm: Par): Task[Unit] =
     Task {
+      print("Evaluating:\n")
       PrettyPrinter.prettyPrint(normalizedTerm)
-      print("\n> ")
     }
 
-  private def repl =
+  private def repl = {
+    val interp = Reduce.makeInterpreter
```

Comment:
Another frog war. I've merged in my PR so now you explicitly have to pass the Tuplespace in.